### DISCUSSION TITLE
<title> New Languages

### DISCUSSION THREAD

**Discussion by AAdam22** [community/community#46387] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Is it possible to enter or add other languages ​​on GitHub in the future, such as the Arabic language, for example??

**Comment by chris-c-thomas** [comment]
Unfortunately I think you're going to have to rely on browsers and/or extensions to achieve this for now.

The support portal has a few although not Arabic: https://github.blog/changelog/2022-08-17-github-support-portal-is-now-available-in-7-new-languages/

In 2016 they stated English would be the only officially support language: https://github.blog/2010-07-13-github-in-your-language/ - don't think much besides the support portal has changed regarding that.
